host/xtest/xtest_20000.c

```diff
@@ -112,14 +112,14 @@ static const struct xtest_enc_fs_case xtest_enc_fs_cases[] = {
 	XTEST_ENC_FS(1, 4094, META1, BLOCK0, VERSION0, VERSION1),
 	XTEST_ENC_FS(1, 4095, META1, BLOCK0, VERSION0, VERSION1),
 	XTEST_ENC_FS(0, 4097, META1, BLOCK0, VERSION0, VERSION1),
-	XTEST_ENC_FS(0, 4097, META1, BLOCK1, VERSION0, VERSION1),
+	XTEST_ENC_FS(0, 4097, META1, BLOCK1, VERSION1, VERSION1),
 	XTEST_ENC_FS(1, 4098, META1, BLOCK0, VERSION0, VERSION1),
-	XTEST_ENC_FS(1, 4098, META1, BLOCK1, VERSION0, VERSION1),
-	XTEST_ENC_FS(1, 1*4096, META1, BLOCK1, VERSION0, VERSION1),
-	XTEST_ENC_FS(1, 2*4096, META1, BLOCK2, VERSION0, VERSION1),
-	XTEST_ENC_FS(1, 3*4096, META1, BLOCK3, VERSION0, VERSION1),
-	XTEST_ENC_FS(1, 4*4096, META1, BLOCK3, VERSION0, VERSION1),
-	XTEST_ENC_FS(1, 4*4096, META1, BLOCK4, VERSION0, VERSION1),
+	XTEST_ENC_FS(1, 4098, META1, BLOCK1, VERSION1, VERSION1),
+	XTEST_ENC_FS(1, 1*4096, META1, BLOCK1, VERSION1, VERSION1),
+	XTEST_ENC_FS(1, 2*4096, META1, BLOCK2, VERSION1, VERSION1),
+	XTEST_ENC_FS(1, 3*4096, META1, BLOCK3, VERSION1, VERSION1),
+	XTEST_ENC_FS(1, 4*4096, META1, BLOCK3, VERSION1, VERSION1),
+	XTEST_ENC_FS(1, 4*4096, META1, BLOCK4, VERSION1, VERSION1),
```

Comment:
Each time a node or a block is written after a sync the version for that particular element will toggle. If there's some changes in policy on how often or which version we're starting with, these has to be updated in order to be able to address the currently used element.

When wrong version is corrupted in these test no error will be detected as an already invalid element was corrupted.

These have a very deep dependency to the format of the secure storage files.